nix/checks/routers.nix

```diff
@@ -53,29 +75,64 @@
           4 # conference <-> expo
         ];
         scale-network = {
+          base.enable = true;
           router.expo.enable = true;
+          services.frr.enable = true;
+          services.frr.router-id = "10.1.2.3";
+          services.frr.passive-interfaces = [
+            "eth0"
+          ];
+          services.frr.broadcast-interface = [
+            "eth1"
+            "eth2"
+          ];
         };
+        networking.firewall.enable = false;
       };
   };
 
   testScript =
-    { nodes, ... }:
+    { ... }:
     ''
       start_all()
       border.wait_for_unit("multi-user.target")
       conference.wait_for_unit("multi-user.target")
       expo.wait_for_unit("multi-user.target")
 
-      # border can hit both routers
+      # layer 2
+      print("LAYER 2")
+
+      # border can ping both routers
       border.succeed("ping -c 5 10.1.1.2")
       border.succeed("ping -c 5 10.1.2.3")
 
-      # conference should hit neighbors
+      # conference can ping both routers
       conference.succeed("ping -c 5 10.1.1.1")
       conference.succeed("ping -c 5 10.1.3.3")
```

Comment:
can this be 10.1.3.1? It's mostly a sanity thing when testing (each link has a .1 and .2)

Reply:
Thats fine, its just currently set that expo uses .3 for the last octet on all of interfaces: https://github.com/socallinuxexpo/scale-network/blob/f505f8b178d79a2461de3d81a0b2714542e45dca/nix/nixos-modules/routers/expo.nix#L39 so we'd need to update it there as well